### PR TITLE
Portable Account

### PR DESCRIPTION
**Pull request by nugaon**
Addressing web3 login workflow where users can retreive their wallet in a decentralized manner from anywhere by providing a _username_ and a _password_.

### CODE REVIEW

File: text/0059-portable-account.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+- FIP: 59
+- title: Portable Account
+- author: Viktor Levente Tóth, @nugaon
+- status: draft
+- created: 2022-10-04
+
+# Summary
+Addressing web3 login workflow where users can retreive their wallet in a decentralized manner from anywhere by providing a _username_ and a _password_.
+
```
Comment (asabya) [inline text/0059-portable-account.md:9]:
```suggestion
Addressing web3 login workflow where users can retrieve their wallet in a decentralised manner from anywhere by providing a _username_ and a _password_.
```

File: text/0059-portable-account.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+# Context, motivation and guide level explanation
+A wallet is an assymetric keypair that represents an account in Ethereum or in Fair Data Society.
```
Comment (asabya) [inline text/0059-portable-account.md:11]:
```suggestion
A wallet is an asymmetric keypair that represents an account in Ethereum or in Fair Data Society.
```

File: text/0059-portable-account.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- ENS registration (username->public key)
+- portable wallet (wallet seed) upload to Ethereum Swarm
+
+The steps are detailed in the following figure
+
+![FDS Account](../resources/fds-account.png)
+
+## Calculating the location of the portable account
+The ENS is used to map usernames to public keys. From the public key, the account's ethereum address can be derived easily for everyone.
+The Ethereum address is needed to contruct Single Owner Chunk (SOC) address along with a consensual 32 bytes of Topic.
```
Comment (asabya) [inline text/0059-portable-account.md:33]:
```suggestion
The Ethereum address is needed to construct Single Owner Chunk (SOC) address along with a consensual 32 bytes of Topic.
```

File: text/0059-portable-account.md (hunk 1 above)
Comment (nugaon) [reply text/0059-portable-account.md:9]: "decentralized" is a correct spelling as well.

**Review by crtahlin — APPROVED** [review]
Looks good

**Pull request merged**